sdcm/tester.py

```diff
@@ -2287,11 +2287,21 @@ def _get_common_email_data(self) -> dict:
         start_time = format_timestamp(self.start_time)
         config_file_name = ";".join(os.path.splitext(os.path.basename(cfg))[0] for cfg in self.params["config_files"])
         test_status = self.get_test_status()
+        backend = self.params.get("cluster_backend")
+        backend = {"docker": "Docker"}.get(backend, backend.upper())
+        if backend == 'AWS':
+            region_name = self.params.get('region_name')
+        elif backend == 'GCE':
+            region_name = self.params.get('gce_datacenter')
+        else:
+            region_name = ''
```

Comment:
There are more than 2 backends, so, I can suggest to make a choice based on a parameters presence instead:

```python
region_name = self.params.get("region_name") or self.params.get("gce_datacenter")
```